qa/standalone/scrub/osd-scrub-repair.sh

```diff
@@ -14,7 +14,8 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU Library Public License for more details.
 #
-source $CEPH_ROOT/qa/standalone/ceph-helpers.sh
+set -x
+source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
```

Comment:
@asomers 
You are moving code back to workunits, were a few commits back things got moved to standalone.